src/QirRuntime/test/QIR-static/qir-test-output.qs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Testing.QIR.Out {
+
+    open Microsoft.Quantum.Intrinsic;
+
+    function MessageTest(msg: String) : Unit
+    {
```

Comment:
```suggestion
    function MessageTest(msg : String) : Unit {
```

Reply:
Thanks for catching. Sorry for the repeated flaw. I'll try to be more attentive.

Reply:
Fixed.